pyroute2.core/pr2modules/netlink/generic/wireguard.py

```diff
@@ -151,11 +151,11 @@ class wgdevice_peer(nla):
                    ('WGPEER_A_PROTOCOL_VERSION', 'uint32'))
 
         class parse_peer_key(nla):
-            fields = (('key', 's'), )
+            fields = (('key', '32s'), )
 
             def decode(self):
                 nla.decode(self)
-                self['value'] = b64encode(self['value'])
+                self['value'] = b64encode(self['key'])
```

Comment:
We might want to also `.decode()` this, so that the keys are strings, the same way they're accepted in other places.

Reply:
Yep, I'll take a look.

Reply:
Please tell me what you decide on, because that's going to determine how I should handle this on my application's side.